packages/components/tests/dummy/app/templates/components/copy/button.hbs

```diff
@@ -0,0 +1,97 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Copy::Button Component"}}
+
+<Shw::Text::H1>Copy::Button</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Generated element</Shw::Text::H2>
+
+  <Shw::Flex as |SF|>
+    <SF.Item class="shw-component-button-generated">
+      <Shw::Label>Default</Shw::Label>
+      <p class="shw-text shw-text-body" id="targetToCopy">"This is the text that these buttons will copy."</p>
+      <Hds::Copy::Button @text="Copy" @targetToCopy="#targetToCopy" />
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H2>Content</Shw::Text::H2>
+
+  <Shw::Flex as |SF|>
+    <SF.Item @label="Default">
+      <Hds::Copy::Button @text="Copy" @targetToCopy="#targetToCopy" />
+    </SF.Item>
+    <SF.Item @label="Icon only">
+      <Hds::Copy::Button @isIconOnly={{true}} @text="Copy" @targetToCopy="#targetToCopy" />
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H2>Sizes</Shw::Text::H2>
+
+  <Shw::Flex as |SF|>
+    {{#each @model.SIZES as |size|}}
+      <SF.Item @label={{capitalize size}}>
+        <Hds::Copy::Button @text="Copy" @size={{size}} @targetToCopy="#targetToCopy" />
+      </SF.Item>
+    {{/each}}
+    <SF.Item @label="Full width">
+      <Shw::Outliner {{style width="300px"}}>
+        <Hds::Copy::Button @text="Copy" @isFullWidth={{true}} @targetToCopy="#targetToCopy" />
+      </Shw::Outliner>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H2>States</Shw::Text::H2>
+
+  <Shw::Grid @columns={{4}} as |SG|>
+    {{#each @model.SIZES as |size|}}
+      {{#each @model.STATES as |state|}}
+        <SG.Item @label="{{capitalize size}} / {{capitalize state}}">
+          <Hds::Copy::Button
+            @text="Copy"
+            @size={{size}}
+            @icon={{this.icon}}
+            mock-state-value={{state}}
+            @targetToCopy="#targetToCopy"
+          />
+        </SG.Item>
+      {{/each}}
+    {{/each}}
+  </Shw::Grid>
+  {{!-- <Shw::Text::H2>Status</Shw::Text::H2>
```

Comment:
@didoo @KristinLBradley if either of you have ideas for how we could make the status show up correctly, I'm open to ideas. It will currently show the correct status color but not the correct status icon.

Reply:
This was an idea I tried that worked but probably wouldn't want to change the code like this just to get Showcase examples working: https://github.com/hashicorp/design-system/pull/1488/files#r1261492130

Reply:
Yeah, probably better not introduce arguments only for the sake of the showcase. 
I may have an idea, but I have to test it (essentially, at page load replace via JS the value of the symbol reference of the SVG icon in the `hds-copy-button--success` buttons).